sbin/Release.sh

```diff
@@ -18,27 +18,33 @@ npm install
 # OpenJDK<version>_<arch>_<os>_<timestampOrTag>.<extension>
 # Examples: OpenJDK8_x64_Windows_201813060547.zip, 
 #           OpenJDK8_x64_LinuxLH_201813060547.tar.gz, 
-#           OpenJDK10_aarch64_Linux_201813060547.sha256.txt
+#           OpenJDK10_aarch64_Linux_201813060547.tar.gz.sha256.txt
 #           OpenJDKamber_x64_Linux_201813061304.tar.gz	
 for file in OpenJDK*
 do
 #              1)VERSION 2)ARCH         3)OS           4)TS_TAG       5)EXTENSION 
-  regex="OpenJDK([a-zA-Z0-9]+)_([a-zA-Z0-9]+)_([a-zA-Z0-9]+)_([a-zA-Z0-9]+).(tar.gz|sha256.txt|zip)";
+  regex="Open(JDK[a-zA-Z0-9]+)_([a-zA-Z0-9]+)_([a-zA-Z0-9]+)_([a-zA-Z0-9]+).(tar.gz|zip)(.sha256.txt)?";
+  echo "Processing $file";
   if [[ $file =~ $regex ]]; 
   then 
-    VERSION=${BASH_REMATCH[1]};
+#   VERSION=${BASH_REMATCH[1]};
     ARCH=${BASH_REMATCH[2]};
     OS=${BASH_REMATCH[3]};
     TS_TAG=${BASH_REMATCH[4]};
     EXTENSION=${BASH_REMATCH[5]};
-    echo "version:${VERSION} arch: ${ARCH} os:${OS} timestampOrTag:${TS_TAG} extension: ${EXTENSION}"; 
+    SHA_EXT=${BASH_REMATCH[6]};
+    echo "version:${VERSION} arch: ${ARCH} os:${OS} timestampOrTag:${TS_TAG} extension: ${EXTENSION} sha_ext: ${SHA_EXT}"; 
   fi
-  mv $file "OpenJDK${VERSION}_${ARCH}_${OS}_${TS_TAG}.${EXTENSION}"
   if [ "$EXTENSION" == "zip" ]; 
   then
     FILENAME=`cat $file | awk  '{print $2}'`
-    sed -i -e "s/${FILENAME}/OpenJDK${VERSION}_${ARCH}_${OS}_${TS_TAG}.${EXTENSION}/g" $file
-    mv $file "OpenJDK${VERSION}_${ARCH}_${OS}_${TS_TAG}.sha256.txt"
+    sed -i -e "s/${FILENAME}/Open${VERSION}_${ARCH}_${OS}_${TS_TAG}.${EXTENSION}/g" $file
```

Comment:
so VERSION holds the variant as well?  e.g. either JDK or J9 and then the actual number?

Reply:
<img width="540" alt="screen shot 2018-06-15 at 6 55 24 am" src="https://user-images.githubusercontent.com/2836948/41464682-27c0d4fc-7069-11e8-9a53-7d9cf4a6c3a3.png">
<img width="306" alt="screen shot 2018-06-15 at 6 54 39 am" src="https://user-images.githubusercontent.com/2836948/41464683-27d2a34e-7069-11e8-9187-1ad3e2c82fe8.png">

Reply:
From Jenkins configure view:
<img width="510" alt="screen shot 2018-06-15 at 6 53 44 am" src="https://user-images.githubusercontent.com/2836948/41464797-91e4e422-7069-11e8-9a5f-5a08ee148b21.png">

From console output:
<img width="540" alt="screen shot 2018-06-15 at 6 55 24 am" src="https://user-images.githubusercontent.com/2836948/41464820-a8450aa8-7069-11e8-802d-76686ee82455.png">

Reply:
Yes, that is in the configuration of the openjdk_release_tool, so to answer your question, looks like VERSION can be JDK8 or JDK8-OPENJ9, etc. and I should not be defining it in the Release.sh script (and overwriting the env var value).

Eventually, I presume some of what @karianna and @johnoliver are doing will lead to more clean-up/transparency in these jobs/scripts too, but for now I want to get to a point where I can extract myself from making too many changes.

Reply:
Sounds good, thanks for tracking it down.  Agree that version and variant should probably split.